commands/azureCommands/create-registry.ts

```diff
@@ -0,0 +1,133 @@
+
+import * as vscode from "vscode";
+import { ContainerRegistryManagementClient } from 'azure-arm-containerregistry';
+import { ResourceManagementClient, SubscriptionModels } from 'azure-arm-resource';
+import { RegistryNameStatus } from "azure-arm-containerregistry/lib/models";
+import { ResourceGroup } from "azure-arm-resource/lib/resource/models";
+import { AzureCredentialsManager } from '../../utils/azureCredentialsManager';
+
+const teleCmdId: string = 'vscode-docker.createRegistry';
```

Comment:
Telemetry should be added before adding this into production.

Reply:
Added that. @northtyphoon Do you know how to confirm whether the telemetry event went through and how to check it?

Reply:
I don't know. Can you ask Chris Dias?

---
In reply to: [205276996](https://github.com/AzureCR/vscode-docker/pull/22#discussion_r205276996) [](ancestors = 205276996)